### PR TITLE
examples/Game-TrafficRush

### PR DESCRIPTION
**Pull request by Kartik-Sama**
This is a simple game in which you need to avoid obstacles as you move on.

### CODE REVIEW

**Comment by satu0king** [comment]
Very Impressive

File: index.html [hunk 1]
```diff
@@ -9,24 +9,216 @@
...
+    var state = 0;
+    var divider = [];
+    var width;
+    var height;
+    var player_car; 
+    var opp_car = []; //array to hold obstracle objects
+    var gameover = 0; //state of game
+    var score = 0; //variable that keeps chak of score, score = no. of obstracles passed.
+    // To initialize the height and width dependent variables adter canvas loads
+    function globalInitialize()
```
Comment (satu0king) [inline index.html]: There is `canvas.setup()` exactly for this, please use that. We can merge once that change is done.

**Review by satu0king — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Kartik-Sama) [reply index.html]: Please can you clarify whether you are expecting changes to be made in canvas.setup in canvas.js. Are we allowed to alter canvas.js ?
Or is it something else you are expecting.

**Comment by Kartik-Sama** [comment]
I have overwritten canvas.setupFunction as you have suggested.

**Pull request closed without merging**